src/backend/distributed/README.md

```diff
@@ -1722,16 +1722,18 @@ Merge command the same principles as INSERT .. SELECT processing. However, due t
 
 # DDL
 
-DDL commands are primarily handled via the ProcessUtility hook, which gets the parse tree of the DDL command. For supported DDL commands, we always follow the same sequence of steps:
+DDL commands are primarily handled via the citus_ProcessUtility hook, which gets the parse tree of the DDL command. For supported DDL commands, we always follow the same sequence of steps:
 
 1. Qualify the table names in the parse tree (simplifies deparsing, avoids sensitivity to search_path changes)
 2. Pre-process logic
-3. Call original ProcessUtility to execute the command on the local shell table
+3. Call original Postgres ProcessUtility to execute the command on the local shell table
```

Comment:
```suggestion
3. Call original previous ProcessUtility to execute the command on the local shell table
```

because it doesn't have to be Postgres' ProcessUtility